cluster/selfplay/deploy-cpu-player.sh

```diff
@@ -12,9 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#!/bin/bash
-
-source ./common.sh
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source ${SCRIPT_DIR}/../common.sh
```

Comment:
i'm a little concerned by the "SCRIPT_DIR" shenanigans when they're immediately followed by referencing a parent directory.  Seems like having a pwd-independent script that immediately asserts a dependency on its directory hierarchy is working at cross purposes.